src/components/Instructions.component.js

```diff
@@ -5,14 +5,29 @@ import './../styles/instructions.css'
 
 class Instructions extends Component {
   render() {
+    var myitems = this.props.items.filter(function(item) {
+      return item.length > 2
+    })
+
+    myitems = myitems.map(function(item) {
+      if (myitems.indexOf(item) % 2 === 0)
```

Comment:
you don't need to use `indexOf`, and shouldn't since it takes O(n) time instead of being constant time. the [array map docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) show that the second argument of the callback is the `index`